src/timeConverter.js

```diff
@@ -1,11 +1,5 @@
-const bignum = require("bignum");
 const milisecondEpochDiff = 62135596800000;
 
 module.exports = function(timeString) {
-    var time = bignum(timeString);
-    var approximateTimestamp = time
-        .div(10000)
-        .sub(milisecondEpochDiff)
-        .toNumber();
-    return approximateTimestamp;
+    return +timeString / 10000 - milisecondEpochDiff;
```

Comment:
This is incorrect.
Incoming timeString is too long for JavaScript number precision and the result of the calculation will be invalid - therefore failing a correct proof validation.

Reply:
Actually i had no probleem at all.
It is working fine for me. Maybe because of node 10+,  I have published this version and i have all tests successed. (Wopi Validator) and people are using

Reply:
The size of JavaScript numbers didn't change. I had to introduce bignum because it wasn't enough. Maybe you've been lucky and didn't get a number resolution shift in the resulting math. 
I've built this for one of the first WOPI implementations outside Microsoft and it's been validating thousands of wopiproof  tokens daily for 4 years. 
Unless Microsoft changed the epoch date they use to Unix epoch, `+timeString` should fail at least occasionally.